docs/v5.0.0 Breaking Change.md

```diff
@@ -0,0 +1,12 @@
+# v5.0.0 Breaking Change
+This release provides an update for the RedisSessionStateProvider nuget package. As a BREAKING CHANGE, the underlying serialization method has changed. SessionStateItemCollection objects are now treated as an atomic unit. The ability to add custom serialization has been removed. These changes were made for necessary security improvements.
+
+The BinaryFormatter type is insecure and can't be made secure [1]. The previous ASP.NET Session State Provider implementation for Redis relies on BinaryFormatter to serialize session state objects. No drop-in replacement exists for BinaryFormatter which does not require defining a contract for the serialized objects [2]. However, the SessionStateItemCollection type offers a serialization method based on BinaryWriter [3].  BinaryWriter serializes primitive types instead of generic objects [4].  
```

Comment:
>The BinaryFormatter type is insecure and can't be made secure [](http://example.com/codeflow?start=0&length=61)

Consider using a bit softer tone :)
BinaryFormatter was implemented before deserialization vulnerabilities were a well-understood threat category. As a result, the code does not follow modern best practices and hence considered insecure.

Reply:
This is copied directly from the documentation: https://learn.microsoft.com/en-us/dotnet/standard/serialization/binaryformatter-security-guide

Reply:
i see. thanks